frontend/src/components/SessionForms/LoginForm.js

```diff
@@ -23,6 +23,16 @@ function LoginForm() {
     return (e) => setState(e.currentTarget.value);
   };
 
+  const demoLogin = (e) => {
+    e.preventDefault();
+    return dispatch(
+      login({
+        email: "user2@demo.com",
+        password: "password",
+      })
+    );
+  };
```

Comment:
this isn't really a good practice since this would be a shared public account and would let anyone create transactions (or bad transactions) to fill up your database and make you pay money. although, technically anyone can create their own account though to do this, but that's a manual step compared to this.

ideally we'd create fake data in redux that can be rendered on your site instead w/o connecting to your database.